classes/Employee.php

```diff
@@ -318,21 +347,25 @@ public static function employeeExists($email)
     /**
      * Check if employee password is the right one
      *
-     * @param string $passwd Password
+     * @param string $passwordHash Password
+     *
      * @return bool result
      */
-    public static function checkPassword($id_employee, $passwd)
+    public static function checkPassword($idEmployee, $passwordHash)
     {
-        if (!Validate::isUnsignedId($id_employee) || !Validate::isPasswd($passwd, 8)) {
+        if (!Validate::isUnsignedId($idEmployee)) {
```

Comment:
we may  issue issue with https://github.com/PrestaShop/PrestaShop/blob/develop/install-dev/data/db_structure.sql#L691
and new hashing ?